lib/api/core/entrypoints/embedded/index.js

```diff
@@ -358,20 +381,28 @@ class EmbeddedEntryPoint extends EntryPoint {
         url += '?' + queryString;
       }
 
-      if (request.input.jwt) {
+      if (user === null && request.input.jwt) {
         try {
           const
             b64Payload = request.input.jwt.split('.')[1],
             payload = Buffer.from(b64Payload, 'base64').toString('utf8');
           user = JSON.parse(payload)._id;
         }
         catch (err) {
-          this.kuzzle.pluginsManager.trigger('log:warn', `Unable to extract user from jwt token: ${request.input.jwt}`);
+          // do nothing: we don't know anything about the token, it may be
+          // invalid on-purpose for all we know
         }
       }
     }
 
-    this.logger.info(connection.ips[connection.ips.length - 1 - this.config.logs.accessLogIpOffset] +
+    const ip = connection.ips[connection.ips.length - 1 - this.config.logs.accessLogIpOffset]
+      || '-';
+
+    if (user === null) {
+      user = '-';
+    }
+
+    this.logger.info(ip +
```

Comment:
Why don't you use string literals?

Reply:
Because that would be impractical and hard to read in this very specific case, given the amount of variables used to create an access log message.